kobo_highlights/main.py

```diff
@@ -233,4 +234,4 @@ def new_config():
     global config
     global CONFIG_PATH
     config = Config.create_interactively().save_file(CONFIG_PATH)
-    console.print("[green]Configuration file created successfully")
+    console.print("[green]Configuration file created successfully to ", CONFIG_PATH)
```

Comment:
I like this wording better :)
```suggestion
        console.print(
        f"[green]The configuration was successfully saved here:\n[cyan]{CONFIG_PATH}"
    )

```